scripts/build.sh

```diff
@@ -41,9 +41,10 @@ export CGO_ENABLED=0
 
 # Allow LD_FLAGS to be appended during development compilations
 LD_FLAGS="-X main.GitCommit=${GIT_COMMIT}${GIT_DIRTY} $LD_FLAGS"
-# In relase mode we don't want debug information in the binary
+
+# In release mode we don't want debug information in the binary
 if [[ -n "${TF_RELEASE}" ]]; then
-    LD_FLAGS="-X main.GitCommit=${GIT_COMMIT}${GIT_DIRTY} -s -w"
+    LD_FLAGS="-X main.GitCommit=${GIT_COMMIT}${GIT_DIRTY} -X github.com/hashicorp/terraform/terraform.VersionPrerelease= -s -w"
```

Comment:
Oooh, tricky! I didn't know Go's linker could be used for shenanigans like this. :grinning: